cdhweb/pages/management/commands/visual_tests.py

```diff
@@ -24,6 +24,8 @@ def take_snapshots(self, browser):
 
         # homepage
         browser.get("http://localhost:8000/")
+        browser.find_element_by_css_selector("#mainmenu label.open").click()
+        browser.find_element_by_css_selector("input[type=search]").send_keys("digital")
         percy_snapshot(browser, "Home")
```

Comment:
Should this be a new snapshot? Don't we need to see home page nav without search active?

Reply:
my thought was that we see it on every other page, so no